src/lib/address/address.ts

```diff
@@ -46,10 +46,10 @@ export const parseMaybeUrlByChainId: Identity<ParseAddressByChainIdInputArgs> =
           assetId,
           value: parsedUrl.target_address ?? value,
           chainId,
-          ...(parsedUrl.parameters?.amount ?? parsedUrl.parameters?.amount
+          ...(parsedUrl.parameters?.value ?? parsedUrl.parameters?.amount
             ? {
                 amountCryptoPrecision: bnOrZero(
-                  parsedUrl.parameters.amount ?? parsedUrl.parameters.amount,
+                  parsedUrl.parameters.value ?? parsedUrl.parameters.amount,
```

Comment:
derp

Reply:
does amount even exist on these urls? looks like just value from the erc spec: https://eips.ethereum.org/EIPS/eip-681